packages/react-search-ui-views/src/ResultsPerPage.js

```diff
@@ -19,6 +19,15 @@ function ResultsPerPage({ className, onChange, options, value }) {
     ? wrapOption(options.find(option => option === selectedValue))
     : null;
 
+  if (selectedOption && !selectedOption.value) {
```

Comment:
I had a thought about this.

Instead of warning when the correct value is not found, I think we should simply show the value.

Something like this:
```
const selectedOption =
    selectedValue && options.includes(selectedValue)
      ? wrapOption(options.find(option => option === selectedValue))
      : wrapOption(selectedValue);

  if (!options.includes(value)) options = [value, ...options];
```

![latest](https://user-images.githubusercontent.com/1427475/56924042-fc7b7280-6a99-11e9-9c45-e61fa4370d09.gif)

In the example above, the default value is `20`, which is not in the configured list of options, so we simply create an option.

Does that makes sense?

Reply:
Yes, it does. You can see after the first selection, the dynamically added option was removed from the list. The developer may get confused about this behavior. What do you think?

Reply:
That's a fair point.

I believe the responsibility of this component is two-fold:

1. Show the current `resultsPerPage` value
2. Allow a user to change the `resultsPerPage` value

To that end, I do think that it should always show the current resultsPerPage value from state, whether it is a valid option or not.

That being said, that is actually probably outside the scope of this PR. We could address that separately.

Reply:
Sure, Thanks.